tests/integration_test_backups.py

```diff
@@ -27,6 +29,38 @@ def _test_backup_list(self):
         backups = [backup for backup in Backup.please.list()]
         self.assertTrue(len(backups))  # at least one backup here;
 
+    def _test_backup_restore(self, backup_id):
+        backup = Backup.please.get(id=backup_id)
+        instance_name = backup.instance
+        instance = Instance.please.get(name=instance_name)
+        classes_count = len(list(instance.classes))
+
+        test_class = Class(name='testclass')
+        test_class.save()
+        instance.reload()
+        classes_count_after_update = len(list(instance.classes))
+
+        self.assertTrue(
+            classes_count_after_update - classes_count == 1,
+            'There should be only 1 more instance class after new class creation.'
+        )
+
+        # wait for backup to be saved
+        while backup.status != 'success':
```

Comment:
If something will fail - we will never end the tests :) 

So the check here - is that restore is created and running - the whole logic is checked on CORE test no need to repeat that.